_includes/tutorials/creating-first-apache-kafka-streams-application/confluent/markup/dev/run-consumer.adoc

```diff
@@ -0,0 +1,29 @@
+Now that the Kafka Streams application is running, run a command line consumer using the `ccloud` CLI to view the events.
+
+After step 4 of this tutorial, your `ccloud` context should be set to the proper environment, cluster, and API Key (see https://docs.confluent.io/ccloud-cli/current/command-reference/index.html[Confluent Cloud CLI Reference] for additional details).
```

Comment:
This feels out of place.  I would either omit it, or if you want to keep, maybe put it in parantheses at the end of the previous sentence?

Reply:
I do think this could trip people so I want to keep the hint. Moved up in 7cffdf9f